src/redux/slices/videogamesSlice.js

```diff
@@ -2,14 +2,19 @@ import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
 import axios from 'axios';
 import BASE_URL from '../constants';
 
-const getDetails = createAsyncThunk('videogames/getDetails', async (id) => {
-  try {
-    const response = await axios.get(`${BASE_URL}/videogames/${id}`);
-    const data = await response.json();
-    return data;
-  } catch (error) {
-    return error;
+const getDetails = createAsyncThunk('videogames/getDetails', async (id, { getState, rejectWithValue }) => {
+  const state = getState();
+  const response = await axios.get(`${BASE_URL}/videogames/${id}`, {
+    headers: {
+      Authorization: state.user.jwt,
+    },
+  }).catch((error) => error);
+
+  if (response.status === 200) {
+    return response.data;
   }
+
+  return rejectWithValue(response.data);
```

Comment:
Would it make sense to reject the thunk with another value? I'm thinking `response.message` as this is a more descriptive string.